src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Machine {
```

Comment:
The 'Machine' class should be abstract because it represents a general concept that should be implemented by more specific machine types.